extensions/blocks/opentable/restaurant-picker.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, FormTokenField } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __, _n } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useRestaurantSearch from './use-restaurant-search';
+
+const MAX_SUGGESTIONS = 20;
+const embedRegex = /<script type=\'text\/javascript\' src=\'\/\/www.opentable.com\/widget\/reservation\/loader\?[^']+\'><\/script>/;
+
+export default function RestaurantPicker( props ) {
+	const [ input, setInput ] = useState( '' );
+	const restaurants = useRestaurantSearch( input, MAX_SUGGESTIONS );
+	const [ selectedRestaurants, setSelectedRestaurants ] = useState( props.rids || [] );
+
+	const idRegex = /^(\d+)$|\(\#(\d+)\)$/;
+
+	const onChange = selected => {
+		const selectedIds = selected.map( restaurant => {
+			const parsed = idRegex.exec( restaurant );
+			const selectedId = parsed[ 1 ] || parsed[ 2 ];
+
+			return selectedId;
+		} );
+		setSelectedRestaurants( selectedIds );
+		props.onChange && props.onChange( selectedIds );
+	};
+
+	const restaurantNames = restaurants.map(
+		restaurant => restaurant.name + ` (#${ restaurant.rid })`
+	);
+
+	const onSubmit = event => {
+		event.preventDefault();
+		props.onSubmit(
+			isEmpty( selectedRestaurants ) && embedRegex.test( input ) ? input : selectedRestaurants
+		);
+	};
+
+	const formInput = (
+		<FormTokenField
+			value={ selectedRestaurants }
+			suggestions={ restaurantNames }
+			onInputChange={ setInput }
+			maxSuggestions={ MAX_SUGGESTIONS }
+			label={ _n( 'Restaurant', 'Restaurants', selectedRestaurants.length ) }
+			{ ...props }
+			onChange={ onChange }
+		/>
+	);
```

Comment:
This looks a bit off to me 😮 

<img width="414" alt="Screenshot 2020-01-06 at 15 00 58" src="https://user-images.githubusercontent.com/2070010/71826244-66223680-3095-11ea-9629-869d83b11ade.png">

Is it supposed to use a serif font (on TwentyTwenty)?